R/install-github.R

```diff
@@ -9,7 +9,7 @@
 #'   (see below); if both is specified, the values in `repo` take
 #'   precedence.
 #' @param ref Desired git reference. Could be a commit, tag, or branch
-#'   name, or a call to [github_pull()]. Defaults to `"master"`.
+#'   name, or a call to [github_pull()]. Defaults to `"HEAD"`.
```

Comment:
```suggestion
#'   name, or a call to [github_pull()]. Defaults to `"HEAD"`, which means the default branch on GitHub.
```

Reply:
Would it be more correct to say the default branch on the git remote? `HEAD` is a git construct, not really GitHub specific after all...

Reply:
Yeah, sure. I would still say something like "default branch on GitHub" or similar, for the people that OK with GitHub but less familiar with git's internals, and maybe link to https://help.github.com/en/github/administering-a-repository/setting-the-default-branch ?

Reply:
What about:
```
Defaults to `"HEAD"`, which means the default branch on GitHub/git. 
See [here](https://help.github.com/en/github/administering-a-repository/setting-the-default-branch) for more details.
```

Reply:
Well, `HEAD` only means the default branch for git remotes, not in git in general. E.g. if you have a git tree, `HEAD` means the tip of the current branch, not the default one. So:

... which means the default branch on GitHub and for git remotes.